FirebaseVertexAI/Tests/Unit/MockURLProtocol.swift

```diff
@@ -16,12 +16,18 @@ import Foundation
 import XCTest
 
 @available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
-class MockURLProtocol: URLProtocol {
-  static var requestHandler: ((URLRequest) throws -> (
-    URLResponse,
-    AsyncLineSequence<URL.AsyncBytes>?
-  ))?
-
+class MockURLProtocol: URLProtocol, @unchecked Sendable {
+  #if compiler(>=6)
+    nonisolated(unsafe) static var requestHandler: ((URLRequest) throws -> (
+      URLResponse,
+      AsyncLineSequence<URL.AsyncBytes>?
+    ))?
+  #else
```

Comment:
Is this still needed in combination with `@unchecked`?

Reply:
If it is still necessary, you can write this as

```swift
#if compiler(>=6)
nonisolated(unsafe)
#endif
static var requestHandler // ...
```

Reply:
Not needed. Removed.

Reply:
Actually it is needed. I had switched back to Swift 5. And Morgan's solution doesn't work since unlike the C preprocessor, the variable modifiers can't be split across the `#if`.